docs/scripts/_jekyll.sh

```diff
@@ -0,0 +1,18 @@
+# Installing Jekyll
+
+echo "Installing Ruby"
+sudo apt-get install ruby-full build-essential zlib1g-dev
+
+echo '# Install Ruby Gems to ~/gems' >> ~/.bashrc
```

Comment:
What is this script? Does it build the docs?

Also do you need to write to the users bashrc? I'd suggest rather just set the variables for the script and write to .gems in LibreTime root dir (with .gitignore entry).

Reply:
I was trying to get it to, but running the commands in my own terminal didn't produce anything. I included it in case anyone else feels like fixing it later, or if we choose to move to Jekyll with Travis later.

Reply:
Ok. Will add a comment in the merge then

Reply:
Sounds good. Thank you so much for helping me through this.